src/structures/interfaces/TextBasedChannel.js

```diff
@@ -321,7 +321,7 @@ class TextBasedChannel {
       }
       if (messageIDs.length === 0) return new Collection();
       if (messageIDs.length === 1) {
-        await this.client.api.channels(this.id).messages(messageIDs[0]).delete();
+        await this.client.api.channels[this.id].messages(messageIDs[0]).delete();
```

Comment:
If you want to enforce square brackets in the library (as proposed in #5256 which seems to be included as given here?) you need to update the latter here. 

Though I'm personally not sure why that's necessary, to begin with.